<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comment</class>
 <widget class="QDialog" name="comment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Crítica</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit_comentario">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Comentário:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_titulo">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Título:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButton0">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton5">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Pontuação:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_comentar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Comentar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_titulo</tabstop>
  <tabstop>textEdit_comentario</tabstop>
  <tabstop>radioButton0</tabstop>
  <tabstop>radioButton1</tabstop>
  <tabstop>radioButton2</tabstop>
  <tabstop>radioButton3</tabstop>
  <tabstop>radioButton4</tabstop>
  <tabstop>radioButton5</tabstop>
  <tabstop>pushButton_comentar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
